src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -1004,6 +1005,10 @@ def get_image_features(
             image_features = self.visual_projection(pooled_output)
         else:
             image_features = pooled_output
```

Comment:
`return_base_image_embeds` argument was added to replace `use_hidden_state` and `return_projected` arguments. Lines 1003-1007 can be deleted.

You can also delete the use_hidden_state arguments in `OwlViTVisionTransformer` and `OwlViTModel.forward()`

Reply:
In `OwlViTVisionTransformer`, `use_hidden_state` is true by default, on removing this param, should it just be this then?

```
last_hidden_state = encoder_outputs[0]
pooled_output = last_hidden_state[:, 0, :]

pooled_output = self.post_layernorm(pooled_output)
```

Reply:
Yes, exactly!

Reply:
Sure, I'll make the changes. I've been travelling lately hence the delay in making the changes, I will update and push shortly.